TYPO3.Fluid/Classes/TYPO3/Fluid/ViewHelpers/Form/AbstractFormFieldViewHelper.php

```diff
@@ -81,9 +81,14 @@ protected function getNameWithoutPrefix()
         } else {
             $name = $this->arguments['name'];
         }
-        if ($this->hasArgument('value') && is_object($this->arguments['value'])) {
-            if (null !== $this->persistenceManager->getIdentifierByObject($this->arguments['value'])
-                && (!$this->persistenceManager->isNewObject($this->arguments['value']))) {
+        if ($this->hasArgument('value')) {
+            /** @var object $value */
```

Comment:
Is this type-hint docblock really needed? There is the `is_object` check anyway in the condition.